templates/config/controller/deployment.yaml.tpl

```diff
@@ -27,11 +27,11 @@ spec:
         - ./bin/controller
         args:
         - --aws-account-id
-        - "$(AWS_ACCOUNT_ID)"
+        - "${AWS_ACCOUNT_ID}"
         - --aws-region
-        - "$(AWS_REGION)"
+        - "${AWS_REGION}"
         - --enable-development-logging
-        - "$(ACK_ENABLE_DEVELOPMENT_LOGGING)"
+        - "${ACK_ENABLE_DEVELOPMENT_LOGGING}"
```

Comment:
These changes need to be undone. The parentheses are necessary. The curly braces are incorrect for Docker args.

Reply:
I wanted to keep this consistent with what we currently have - https://github.com/aws/aws-controllers-k8s/blob/main/services/ecr/config/controller/deployment.yaml#L34. Thus, I made this change.